src/index.html

```diff
@@ -13,6 +13,52 @@
     />
```

Comment:
The attributes of the <link> tag should be on separate lines with 2-space indentation for consistency and readability, as per the checklist guidelines.